clientgui/DlgAdvPreferences.h

```diff
@@ -28,35 +28,36 @@
 #include "cc_config.h"
 
 class CDlgAdvPreferences : public CDlgAdvPreferencesBase {
-	DECLARE_DYNAMIC_CLASS( CDlgAdvPreferences )
+    DECLARE_DYNAMIC_CLASS( CDlgAdvPreferences )
     DECLARE_EVENT_TABLE()
-	void ReadPreferenceSettings();
-	bool SavePreferencesSettings();
-	void UpdateControlStates();
-	void SetSpecialTooltips();
+    void ReadPreferenceSettings();
+    bool SavePreferencesSettings();
+    void UpdateControlStates();
+    void SetSpecialTooltips();
     bool SaveState();
     bool RestoreState();
-	bool ValidateInput();
-	void SetValidators();
-	bool IsValidFloatChar(const wxChar& ch);
-	bool IsValidFloatValue(const wxString& value, bool allowNegative=false);
+    bool ValidateInput();
+    void SetValidators();
+    bool IsValidFloatChar(const wxChar& ch);
+    bool IsValidFloatValue(const wxString& value, bool allowNegative=false);
     bool IsValidFloatValueBetween(const wxString& value, double minVal, double maxVal);
-	bool IsValidTimeChar(const wxChar& ch);
-	bool IsValidTimeValue(const wxString& value);
-	void ShowErrorMessage(wxString& msg,wxTextCtrl* errorCtrl);
-	bool EnsureTabPageVisible(wxTextCtrl* txtCtrl);
-	bool ConfirmClear();
-	wxString DoubleToTimeString(double dt);
-	double TimeStringToDouble(wxString timeStr);
+    bool IsValidTimeChar(const wxChar& ch);
+    bool IsValidTimeValue(const wxString& value);
+    void ShowErrorMessage(wxString& msg,wxTextCtrl* errorCtrl);
+    bool EnsureTabPageVisible(wxTextCtrl* txtCtrl);
+    bool ConfirmClear();
+    bool ConfirmSetLocal();
```

Comment:
I think only this line is added to this document. other changes are because of different settings for tabulations and blanks used on your environment. Please don't blame me that I'm picky but could you please revert unnecessary changes from this file?

Reply:
@JuhaSointusalo asked me, in effect, to ensure that my own changes complied with BOINC's  [Coding Style](https://boinc.berkeley.edu/trac/wiki/CodingStyle#indent). I've seen other committers clean up other files when they happen to revisit them, so it seemed 'the thing to do' while I was here, but I can remove those changes if people feel I've mis-interpreted the policy.

Reply:
When code and whitespace changes are mixed together it becomes harder to see the code changes. So it's usually better to do whitespace only changes separately. If the lines you are editing have wrong indentation then of course fix the indentation too.

I could have said something about it but in trying to balance between relaxed, everything goes reviewer and, the more typical of me, hard-ass nitpicker I decided to give you some slack. This is a somewhat small PR and the whitespace changes are fairly isolated from code changes. Since the whitespace changes are already in I can live with them continuing to be here.

If you do remove the whitespace changes could you squash the commits or make a new, clean PR? Otherwise this gets a bit silly.